.github/workflows/docker-debian.yml

```diff
@@ -0,0 +1,139 @@
+name: Docker Images Debian
+
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - v*
+  pull_request:
+    branches:
+    - main
+  schedule:
+    - cron:  '23 1 * * 0'
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  docker:
+    name: Build and publish Debian images
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Prepare
+        id: prep
+        run: |
+          DOCKER_IMAGE=madanalysis5/madanalysis5
+          VERSION=latest
+          MADANALYSIS_VERISON=v1.10.0
```

Comment:
The version number will need to get bumped during releases (might want to look into things like `bump2version` for the whole project to make this easier/automatic).

Reply:
Thanks @matthewfeickert I'll definitely look into this, it makes total sense to automatize the versioning.